daemon/cluster/services.go

```diff
@@ -50,14 +50,16 @@ func (c *Cluster) GetServices(options apitypes.ServiceListOptions) ([]types.Serv
 		return nil, err
 	}
 
+	if len(options.Filters.Get("runtime")) == 0 {
+		// Default to using the container runtime filter
```

Comment:
I would prefer the CLI, but I didn't want to make a breaking change to the API here.